### PR TITLE
Little details

### PR DESCRIPTION
**Pull request by elrumordelaluz**
1. Unify keyframes with same value
1. Unify `size` props (in case will update [`README`](https://github.com/skratchdot/react-github-corner/blob/master/README.md)). Adds also a `PropTypes.oneOfType` to avoid warning when using a `String` and match the [Documentation](https://github.com/skratchdot/react-github-corner#documentation)
1. Adds `style` prop to pass and apply to the main `SVG` allowing to make the _octocat_ transparent, for example
    
    ```
    <GithubCorner href={githubUrl}  style={{ 'mixBlendMode': 'darken' }}/>
    ```
    More _use-cases_ in [this issue](https://github.com/tholman/github-corners/issues/15) from the original repo

### CODE REVIEW

**Comment by skratchdot** [comment]
Looks good to me. Thanks for taking an interest and the time to submit a PR!

The `size` prop breaks backwards compatibility, so I will need to bump a major version for this, but it seems okay to me.  Is the size change absolutely necessary though?  If it is, we need to fix the examples on the homepage (things like: https://github.com/skratchdot/react-github-corner/blob/da73ec4f591e236ebf685c2a05d539acb08660c4/src/app/pages/Home.js#L72-L73).

We also need to fix the README: https://github.com/skratchdot/react-github-corner/blame/master/README.md#L49-L50

Basically, anywhere that width/height exist in this repo would need to be fixed.

**Comment by elrumordelaluz** [comment]
Will _update_ each place where reference `width` and `height` into `size`.

**Comment by skratchdot** [comment]
Awesome!  Thanks for making these additional changes.

**Comment by skratchdot** [comment]
@elrumordelaluz - One thing I noticed after pulling this down and running locally: i had a test that was using "style" to set the style on the outside anchor tag.  I think I'm going to rename the new "style" property you added to "svgStyle".  Once I do that, I'll bump the major version and publish this on npm.  Thanks again.

**Comment by skratchdot** [comment]
Just published version `2.0.0`: https://www.npmjs.com/package/react-github-corner
